forte/data/ontology/core.py

```diff
@@ -424,11 +426,50 @@ def as_pointer(self, from_entry: "Entry") -> "Pointer":
                 "Do not support reference a multi pack entry from an entry."
             )
 
+    def version_older_than(self, pack_version, compare_version) -> bool:
+        """
+        Check if the version of pack is older than the compare_version
+
+        Args:
+            pack_version: The version to be examined.
+            compare_version: The standard version number to be compared.
+
+        Returns:
+            bool indicating compare result.
+        """
+        if pack_version[0] < compare_version[0]:
+            return True
+        elif pack_version[0] > compare_version[0]:
+            return False
+        else:  # equal
+            if pack_version[1] < compare_version[1]:
+                return True
+            elif pack_version[1] > compare_version[1]:
+                return False
+            else:  # equal
+                if pack_version[2] < compare_version[2]:
+                    return True
+                elif pack_version[2] >= compare_version[2]:
+                    return False
```

Comment:
Is it possible that we first make sure the version is a len 3 list and compare them in one shot? https://docs.python.org/3/tutorial/datastructures.html#comparing-sequences-and-other-types